.gitattributes

```diff
@@ -1,5 +1,5 @@
 # Set the default behavior, in case people don't have core.autocrlf set.
-* text=auto
+* text=auto eol=lf
 
 # Explicitly declare text files you want to always be normalized and converted
 # to native line endings on checkout.
```

Comment:
Update the comment such that it reflects that we don't want the native line endings but uniformly `lf`

```suggestion
# Explicitly declare text files you want to always be normalized and converted
# to lf line endings on checkout.
```